pysolr.py

```diff
@@ -708,10 +708,7 @@ def _from_python(self, value):
             else:
                 value = "%sT00:00:00Z" % value.isoformat()
         elif isinstance(value, bool):
-            if value:
-                value = "true"
-            else:
-                value = "false"
+            value = "true" if value else "false"
```

Comment:
I would disable this. It’s harder to read, generates more diff noise if you change either branch, and vertical height isn’t a strictly limited resource.